metricshub-doc/src/site/markdown/release-notes.md

```diff
@@ -5,6 +5,81 @@ description: Learn more about the new features, changes and improvements, and bu
+| [**\#102**](https://github.com/MetricsHub/enterprise-connectors/issues/102) | **RedFish (REST)**: Connector fails to request REST API URLs                                                                                                                                                                                                           |
+| [**\#109**](https://github.com/MetricsHub/enterprise-connectors/issues/109) | **Brocade SAN Switch**: The `bandwidth` attribute incorrectly displays the WWN instead of the actual network bandwidth                                                                                                                                                 |
+| [**\#104**](https://github.com/MetricsHub/enterprise-connectors/issues/104) | **HPE OneView (Frames and Blades)**: The CPU collect fails due to a missing REST API URL                                                                                                                                                                               |
+| [**\#106**](https://github.com/MetricsHub/enterprise-connectors/issues/106) | **Palo Alto Firewall (SNMP)**: The `firewall.sessions` metric reports the protocol attribute as `ucp` instead of `udp`                                                                                                                                                 |
+
+### MetricsHub Community Edition v1.0.08
+
+#### What's New
+
+| ID                                                                         | Description                                                                                                                                                                                                                               |
+| -------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**\#765**](https://github.com/MetricsHub/metricshub-community/issues/765) | Added [MCP tool](./integrations/ai-agent-mcp.md) to get host details                                                                                                                                                                      |
+| [**\#772**](https://github.com/MetricsHub/metricshub-community/issues/772) | Custom monitoring jobs in `metricshub.yaml` can now reference resource attributes using the syntax `${esc.d}{resource.attribute::ATTRIBUTE-KEY}`, where ATTRIBUTE-KEY is configured at the resource level                                 |
```

Comment:
@NassimBtk : can you provide more information about value the issues https://github.com/MetricsHub/metricshub-community/issues/772 and https://github.com/MetricsHub/metricshub-community/issues/773 bring to the final user. 

The RN entry is really technical and I could not find any information about this dev in the user doc.

Reply:
Yes, it’s a technical aspect designed exclusively for connector developers. If the release note should target only non-developer users, we can remove these two entries. 
This will be documented in https://metricshub.org/community-connectors/develop/references.html

Reply:
@NassimBtk , if these new features are:
- not documented yet, I would rather remove these entries from the RN
- documented, maybe:
     -  prefix the RN entry so that readers can quickly identify if they are concerned or not (ex: [Connector Development], [Connector Design], etc.) 
     - simplify the description (ex: Added the ability reference resource attributes when configuring custom monitoring jobs)
     - link the releasee notes entry to the documentation for further information.